package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Encore does a lot of work under the hood for us. This includes compiling down to certain js versions if i remember correctly.

We can configure this ourselves by adding a browserlist configuration in the package.json. It may be worthwile to find out what the defaults are, and if those align with the needs of this project.

Reply:
Thanks for the reminder. I offloaded this to a new ticket. For now supporting ancient browsers is not a requirement. And the out of the box settings are probably good enough.